dstep/driver/Application.d

```diff
@@ -34,30 +38,34 @@ class Application : DStack.Application
 
     private
     {
-        string[] inputFiles;
-
-        Index index;
-        TranslationUnit translationUnit;
-        DiagnosticVisitor diagnostics;
-
+        bool helpFlag;
         Language language;
         string[] argsToRestore;
-        bool helpFlag;
         Compiler compiler;
     }
 
     protected override void run ()
     {
         handleArguments;
-
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            auto inputFiles = arguments.argument.input.values;
+            bool createNewFileName = inputFiles.length > 1 ? true : false;
+            foreach(string fileName; inputFiles)
```

Comment:
Please have an empty newline before and after statements like if, foreach and so on.